nepalingo-web/src/components/GreetingCard.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import ReactGA from 'react-ga4';
+
+interface GreetingCardProps {
+
+  name: string;
+}
+
+const cardStyle: React.CSSProperties = {
+  border: '2px solid #000',
+  borderRadius: '15px',
+  padding: '20px',
+  margin: '10px',
+  textAlign: 'center',
+  backgroundColor: '#f0f0f0',
+  boxShadow: '0 4px 8px rgba(0, 0, 0, 0.2)',
+  fontFamily: 'Arial, sans-serif',
+};
+
+const headingStyle: React.CSSProperties = {
+  fontSize: '24px',
+  color: '#333',
+};
+
+const GreetingCard: React.FC<GreetingCardProps> = ({ name }) => {
+  ReactGA.send({ hitType: "pageview", page: window.location.pathname, title: "home" });
+
+  const getCurrentGreeting = (): string => {
+    const CurrentHours = new Date().getHours();
+    if (CurrentHours < 12) {
+      return 'Good Morning';
+    }
+    else if (CurrentHours < 18) {
+      return 'Good Afternoon';
+    } else {
+      return 'Good Evening';
+    }
+  }
+  return (
+    <div style={cardStyle}>
+      <h1 style={headingStyle}>{`${getCurrentGreeting()}, ${name}!`}  {name}</h1>
```

Comment:
Awesome work on the if statement! It looks like it's working!
You have name here twice accidentally though 😂
![CleanShot 2024-07-17 at 22 19 14@2x](https://github.com/user-attachments/assets/171d5695-eaa4-46d0-b1d5-2e05b1dcfb5b)